src/index.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html>
+<head>
+    <title>Idea Town</title>
+    <!-- inject:css -->
```

Comment:
possibly dumb question... are we using something which will actually inject css and js with these comment blocks, or is this just for us as a mental note?